.github/workflows/pd-tests.yaml

```diff
@@ -16,13 +16,11 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        worker_id: [1, 2, 3, 4]
-    outputs:
-      job-total: ${{ strategy.job-total }}
+        worker_id: [1, 2, 3, 4, 5, 6, 7, 8]
```

Comment:
why the work_id is  different between pd-cover and pd-tests ?

Reply:
`pd-cover` does not enable `race` or `deadlock` so it can run faster. From the test time cost, maybe, change to `pd-cover: 4, pd-tests: 6` is more suitable